siriuspy/siriuspy/injctrl/main.py

```diff
@@ -1162,6 +1163,7 @@ def _wait_topup_period(self):
                     curr_avg=self._target_current,
                     curr_now=self.currinfo_dev.current,
                     ltime=self.currinfo_dev.lifetime)
+                self._update_log(f'BiasFB Ideal InjCurr: {dcur:.3f}mA')
```

Comment:
@anacso17 , consider replacing the word "Ideal" by the word "required".

Reply:
done!